Makefile

```diff
@@ -699,6 +699,27 @@ test-helm-update-snapshots: helmunit/installed
 .PHONY: test-go
 test-go: test-go-prepare test-go-unit test-go-libfido2 test-go-touch-id test-go-tsh test-go-chaos
 
+#
+# Runs a test to ensure no environment leak in build, called by CI/CD.
```

Comment:
I don't see this being called anywhere.

Reply:
The workflow could execute this, but the failure cause may be less obvious.  So we did not use this directly in the workflow.

I wanted to keep this though.  I think it's important that test can be run locally

Reply:
Ok, let's call it `test-env-leakage` or something then. `test-env` sounds like some sort of generic "prepare a test environment" step.

and remove the `called by CI/CD` comment since it is not true :-)

Reply:
Done, thank you!

Reply:
Mind updating the PR title too (so we don't get an inaccurate commit message)?